catalyst-gateway/Cargo.toml

```diff
@@ -48,4 +48,5 @@ panic = "deny"
 string_slice = "deny"
 unchecked_duration_subtraction = "deny"
 unreachable = "deny"
-missing_docs_in_private_items = "deny"
\ No newline at end of file
+missing_docs_in_private_items = "deny"
+arithmetic_side_effects = "deny"
```

Comment:
This change has to be made in `catalyst-ci` or it will fail CI checks for consistent configuration.

Reply:
I understand that. It is more like an invitation for discussion at this point. I will proceed with proper implementation.

Reply:
I suggest making a draft branch with the necessary change in cat-ci.  You use the branch name then, instead of a cat-ci version in the earthly file.  It will allow CI to complete.  Would be good to see this PR pass CI.

Reply:
I have updated the pull request and it seems working. Here is the `catalyst-ci` branch: https://github.com/input-output-hk/catalyst-ci/pull/365